core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -522,6 +522,7 @@ FlowIdentifier updateSaltByName(@PathParam("workspaceId") Long workspaceId, @Pat
     @ApiOperation(value = DATABASE_BACKUP, nickname = "databaseBackup")
     BackupV4Response backupDatabaseByName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name,
             @QueryParam("backupLocation") String backupLocation, @QueryParam("backupId") String backupId,
+            @QueryParam("skipDatabaseNames") List<String> skipDatabaseNames,
```

Comment:
general question: why do you decided to pass this as a method parameter instead of a parameter in application.yml? Is it expected to change a lot?

Reply:
Yes, it's currently used only in the context of a backup occurring before the upgrade. without this the backup has no other information to understand the context it's being called in.